index.html

```diff
@@ -0,0 +1,172 @@
+              </div>
+            </div>
+          </li>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="buttons">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavScript</li>
+                <li>html</li>
+              </ul>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="buttons">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavScript</li>
+                <li>html</li>
+              </ul>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <div class="about-me">
+      <section class="about-container">
+        <div class="about-header">
+          <h2>About me</h2>
+        </div>
+        <div class="about-style">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have
+            a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="get-resume-button">
+          <button type="button">Get My Resume</button>
+        </div>
+        <div class="skills-container">
+          <ul>
+            <li class="item-list">
+             
+              <div>
+                <h2>Languages</h2>
+              </div>
+              <div class="about-buttons">
+                <div>Ruby on Rails</div>
+                <div>Ruby</div>
+                <div>Html</div>
+                <div>CSS</div>
+              </div>
+            </li>
+
+            <li class="item-list">
+             
+              <div>
+                <h2>Frameworks</h2>
+              </div>
+              <div class="about-buttons">
+                <div>Bootstrap</div>
+                <div>Ruby on rails</div>
+                <div>RSpec</div>
+                <div>Capybara</div>
+                <div>Selenium</div>
+              </div>
+            </li>
+            <li class="item-list">
+              
+              <div>
+                <h2>Skills</h2>
+              </div>
+              <div class="about-buttons">
+                <div>Codekit</div>
+                <div>Github</div>
+                <div>Codepen</div>
+                <div>Gitlab</div>
+                <div>Terminal</div>
+              </div>
+            </li>
+          </ul>
```

Comment:
- [ ] Kindly add missing icons for each card following the Figma design
- Figma
<img width="244" alt="Screen Shot 2022-03-26 at 09 01 35" src="https://user-images.githubusercontent.com/28307960/160228771-27923219-1275-42c4-bce4-34212489c871.png">

- Your design
<img width="364" alt="Screen Shot 2022-03-26 at 08 59 41" src="https://user-images.githubusercontent.com/28307960/160228758-7a625c0a-e95e-419f-b21a-1135b36837eb.png">

Reply:
thank you already fixed